due-processed/sass/site/_site-footer.scss

```
@@ -4,7 +4,72 @@
 }
 
 .site-info {
-	@include primary-page-width();
-	padding-top: $spacing__large;
-	padding-bottom: $spacing__large;
+	width: calc(100% - (2 * 60px));
```

I think you can use the mixin and just define what's different:
```suggestion
	@include primary-page-width();
	max-width: 1094px; // this looks like the only property that needs to be overridden
```
then can also remove the 
```
@include breakpoint-below(tablet-portrait) {
		width: calc(100% - (2 * 20px));
```
section below

---

That causes an error: `Unexpected duplicate "max-width"  declaration-block-no-duplicate-properties`

---

let's just turn that rule off, I think it's better to duplicate `max-width` than to duplicate the mixin. Add this line to `.stylelintrc.json`:
```
"declaration-block-no-duplicate-properties": null
```